site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -397,9 +395,8 @@
 						<tr>
 							<td>
 								<span id ="group$g.Id">$formattedText.escapeHtml($g.Title)</span>
-
 							</td>
-							#if ($!viewRoster || $!viewMembershipGroups.contains($g.Id))
+							#if ($!viewRoster)
```

Comment:
Should be the if inside the <td>?

Reply:
No, because we want to leave that column out entirely if the current user can't view rosters.